src/Native/Runtime/inc/eetype.h

```diff
@@ -444,6 +444,11 @@ class EEType
     bool IsGeneric()
         { return (m_usFlags & IsGenericFlag) != 0; }
 
+#if defined(CORERT)
+    ModuleManager* GetModule()
```

Comment:
There are too many different existing things called module. I know that ModuleManager is not a great name ... I would love to have a better name, but I think it should be something more specific than Module.